packages/smooth_app/lib/pages/preferences_v2/app_bars/search_bottom_bar.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:smooth_app/generic_lib/design_constants.dart';
+import 'package:smooth_app/generic_lib/widgets/smooth_text_form_field.dart';
+import 'package:smooth_app/pages/preferences_v2/roots/preferences_root.dart';
+import 'package:smooth_app/themes/smooth_theme.dart';
+import 'package:smooth_app/themes/smooth_theme_colors.dart';
+
+const double SEARCH_BOTTOM_HEIGHT = 86.0;
+
+class SearchBottomBar extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final ThemeData theme = Theme.of(context);
+    final SmoothColorsThemeExtension themeExtension =
+        context.extension<SmoothColorsThemeExtension>();
+
+    return SizedBox(
+      height: SEARCH_BOTTOM_HEIGHT,
+      child: CustomPaint(
+        painter: _BottomPainter(
+          color: themeExtension.primaryMedium,
+          radius: ROUNDED_RADIUS,
+        ),
+        child: Padding(
+          padding: const EdgeInsetsDirectional.only(
+            top: LARGE_SPACE * 2,
+            start: MEDIUM_SPACE,
+            end: MEDIUM_SPACE,
+            bottom: MEDIUM_SPACE,
+          ),
+          child: Row(
+            children: <Widget>[
+              Expanded(
+                child: SmoothTextFormField(
+                  type: TextFieldTypes.PLAIN_TEXT,
+                  controller: TextEditingController(),
+                  hintText: 'Rechercher un paramètre (ex: NutriScore)',
```

Comment:
```suggestion
                  hintText: 'Rechercher un paramètre (ex: Nutri-Score)',
```

Reply:
Would rather make sense in `app_en.arb` btw.

Reply:
ping